client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -3,30 +3,67 @@
  */
 import React, { Component, PropTypes } from 'react';
 import classNames from 'classnames';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import Main from 'components/main';
 import Card from 'components/card';
-import SectionHeader from 'components/section-header';
+import { getSelectedSite } from 'state/ui/selectors';
+import Main from 'components/main';
+import Setup from './setup';
 
-export default class Dashboard extends Component {
+class Dashboard extends Component {
 
 	static propTypes = {
 		className: PropTypes.string,
 	};
 
-	render() {
+	onStoreSetupFinished = () => {
+		// TODO - save that setup has been finished to the store's state on WPCOM
+		// TODO - is there a way to set an option on the store site?
+	}
+
+	renderStoreSetup = () => {
+		const { selectedSite } = this.props;
+		return (
+			<Setup
+				onFinished={ this.onStoreSetupFinished }
+				site={ selectedSite }
+			/>
+		);
+	}
+
+	renderStoreManagement = () => {
+		return (
+			<Card>
+				<p>This is the start of something great!</p>
+				<p>This will be the home for your WooCommerce Store integration with WordPress.com.</p>
+			</Card>
+		);
+	}
+
+	render = () => {
+		const { storeSetupCompleted } = this.props;
+
 		return (
 			<Main className={ classNames( 'dashboard', this.props.className ) }>
-				<SectionHeader label="WooCommerce Store" />
-				<Card>
-					<p>This is the start of something great!</p>
-					<p>This will be the home for your WooCommerce Store integration with WordPress.com.</p>
-				</Card>
+				{
+					storeSetupCompleted ? this.renderStoreManagement() : this.renderStoreSetup()
+				}
 			</Main>
 		);
 	}
 
 }
+
+function mapStateToProps( state ) {
+	// TODO - figure out from state if setup has been completed for this store yet
+
+	return {
+		selectedSite: getSelectedSite( state ),
+		storeSetupCompleted: false
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14202/commits/66d13147f190def3689b815188fde46da04a36b4